oidc.go

```diff
@@ -246,6 +246,15 @@ func (p *Provider) UserInfo(ctx context.Context, tokenSource oauth2.TokenSource)
 	return &userInfo, nil
 }
 
+func mediaType(resp *http.Response) string {
```

Comment:
You want mime.ParseMediaType

https://play.golang.org/p/7hDblPWcorU

```go
func mediaType(resp *http.Response) string {
    ct := resp.Header.Get('Content-Type")
    if t, _, err := mime.ParseMediaType(ct); err == nil {
        return t
    }
    return ""
}
```

Reply:
Thanks. Updated.